pkg/storaged/side-panel.jsx

```diff
@@ -32,12 +32,28 @@ export class SidePanel extends React.Component {
     constructor() {
         super();
         this.state = { collapsed: true };
+        this.current_rows_keys = [];
+        this.new_keys = [];
     }
 
     render() {
         let show_all_button = null;
-        let rows = this.props.rows;
+        let rows = this.props.rows.filter(!!row);
+
+        // Find new items for animations
+        const current_keys = [];
+        rows.forEach(row => { current_keys.push(row.key) });
```

Comment:
In case you have to push again: This is much nicer as a map operation:

    const current_keys = rows.map(row => row.key);

Reply:
Of yeah, I think I had there also some condition before and ha would lead o `.reduce()` but now this surely can be done with `map()`. I'll update

Reply:
done